### PR TITLE
Add changelog for 6.5.0

### PR DESCRIPTION
**Pull request by simitt**

### CODE REVIEW

File: changelogs/6.5.asciidoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[[release-notes-6.5]]
+== APM Server version 6.5
+
+https://github.com/elastic/apm-server/compare/6.4\...6.5[View commits]
+
+* <<release-notes-6.5.0>>
+
+[[release-notes-6.5.0]]
+=== APM Server version 6.5.0
```
Comment (jalvz) [inline changelogs/6.5.asciidoc]: did we stop doing sections? (added, bug fixes, etc.)

File: changelogs/6.5.asciidoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[[release-notes-6.5]]
+== APM Server version 6.5
+
+https://github.com/elastic/apm-server/compare/6.4\...6.5[View commits]
+
+* <<release-notes-6.5.0>>
```
Comment (jalvz) [inline changelogs/6.5.asciidoc]: Need to refer this in `docs/release-notes.asciidoc`

File: changelogs/6.5.asciidoc (hunk 1 above)
Comment (simitt) [reply changelogs/6.5.asciidoc]: good question, we didn't do it for `6.4.`, but I have no strong feelings, if preferred we can add them.

Comment (jalvz) [reply changelogs/6.5.asciidoc]: yes, i saw that. i don't know if it was overlooked or dropped on purpose. i think it helps

Comment (graphaelli) [reply changelogs/6.5.asciidoc]: +1 on adding those sections

**Pull request merged**